templates/wordpress/index.ts

```diff
@@ -4,32 +4,59 @@ import { Input } from "./meta";
 export function generate(input: Input): Output {
   const services: Services = [];
   const mysqlPassword = randomPassword();
+  const redisPassword = randomPassword();
+
+  const appEnv = [
+    `WORDPRESS_DB_HOST=${input.projectName}_${input.databaseServiceName}`,
+    `WORDPRESS_DB_USER=${input.databaseType}`,
+    `WORDPRESS_DB_PASSWORD=${mysqlPassword}`,
+    `WORDPRESS_DB_NAME=${input.projectName}`,
+  ];
+
+  const fpm = input.appServiceImage.includes("fpm");
+  const appPort = fpm ? 9000 : 80;
+  const exec = fpm ? "php-fpm" : "apache2-foreground";
+  const alpine = input.appServiceImage.includes("alpine")
+    ? "apk add $PHPIZE_DEPS && "
+    : "";
+  const cmd =
+    alpine +
+    "pecl install redis && docker-php-ext-enable redis && docker-entrypoint.sh " +
```

Comment:
This is a smart solution, but I don't want to have people install things on every service restart. Ideally, this should be a separate docker image or a github repo with a Dockerfile. I'm happy to host it on `https://github.com/esaypanel-io/wordpress-redis` if you want to build it.

Reply:
Actually, a small change is enough [link](https://github.com/berksmbl/wordpress-redis)